lib/_http_client.js

```diff
@@ -504,19 +504,17 @@ ClientRequest.prototype._deferToConnect = function(method, arguments_, cb) {
   // in the future (when a socket gets assigned out of the pool and is
   // eventually writable).
   var self = this;
+  var callSocketMethod = function() {
+    if (method) {
+      self.socket[method].apply(self.socket, arguments_);
+    }
+    if (cb) { cb(); }
```

Comment:
Yes, core prefers to leave out braces in `if` statements. If there are any `else` conditions that require braces, then all branches use braces for consistency (not applicable here).